journal/modernized/02-chapter-1.adoc

```diff
@@ -1,6 +1,8 @@
-== Journal of the Life of William Edmundson.Section I.
+== Journal of the Life of William Edmundson. Section I.
+//Instead of this saying "secton I." do you want it to say Chapter 1? And no Roman numeral? Or are these ones okay?
 
 Giving an account of his birth, parentage, trade, marriage, convincement, etc.
+//^^Does this need to have any subtitle syntax or whatever? It seems to be summarizing the chapter, but I couldn't find anything that fit it in the style guide. Maybe a secondary heading? Idk.
```

Comment:
I just noticed it's italicized in the proofread copy....is italicizing it the solution? I feel like a helpless infant working on this...lol

Reply:
Don't worry! There is a lot to learn in the beginning, but you're a quick learner and this will be a breeze after a bit. :)